app/controllers/volunteers_controller.rb

```diff
@@ -66,6 +66,37 @@ def remove
     # for volunteer to remove project, perhaps include a note for why they're
     # no longer available?
   end
+  
+  def add_repairs
+    @repairs = Repair.all
+    # @repairs = Repair.find(params[:id])
+    @volunteer = current_user.volunteer
+  end
+  
+ def add_repair_to_volunteer
+  # @repair = Repair.find(params[:id])
+   volunteer_repair = VolunteerRepair.new
+   volunteer_repair.volunteer_id = current_user.volunteer.id
```

Comment:
Again, current_user.volunteer might be nil, in which case this will throw an error.  You need to check for that case and redirect to volunteer new